src/main.rs

```diff
@@ -481,8 +543,19 @@ impl HabitCtl {
             .collect();
         done.retain(|value| *value);
 
+        let mut skip: Vec<bool> = self
+            .habits
+            .iter()
+            .map(|habit| {
+                let status = self.day_status(&habit, &score_date);
+                habit.every_days > 0
+                    && (status == DayStatus::Skipped || status == DayStatus::Skipified)
+            })
+            .collect();
```

Comment:
If I'm correct, in Python-ish pseudocode this would be `[habit.every_days > 0 & (habit.status() in [Skipped, Skipified] for habit in habits]`, right? Looks reasonable!

Reply:
Much the same as I did for the skipified function above, the let mut done function was copied here and the functions modified to support skips.to yield a correct score if there are skips (basically, allow us to subtract skips from the denominator via the skips.len call.